### PR TITLE
Upgrade [ch21443]

### PR DESCRIPTION
**Pull request by pkopac**
* turns out tests were not cleaned up
* reverting the package.json to latest versions for devDeps
* build with Go 1.14

### CODE REVIEW

File: webui/internal/assets/package.json [hunk 1]
```diff
@@ -8,31 +8,31 @@
     "cover": "istanbul cover _mocha --root src/ --include-all-sources -x '**/*.test.js' -x 'index.js' -- 'src/**/*.test.js' --require babel-register --require ignore-styles; exit 0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.0.6",
+    "babel-core": "latest",
```
Comment (ikanade) [inline webui/internal/assets/package.json]: Why can't we lock this to recent most version? `latest` could install a package which might have breaking change for us.

Comment (pkopac) [reply webui/internal/assets/package.json]: You have a good point. There are some GUI tests in the assets folder, I'll attempt to enable them on Travis, so we'll see whether it's breaking outright or not.

**Comment by pkopac** [comment]
@ikanade the UI build process is a bit complicated, but I think I figured out correct versions and found the approach, so it should now work :crossed_fingers:

**Comment by pkopac** [comment]
I hope one lucky green build is good enough :joy: the tests are somewhat time-dependent. I'd rather not spend any more time on this.

**Pull request merged**